build/docker/zap-baseline.py

```diff
@@ -469,18 +536,45 @@ def main(argv):
         with open(base_dir + report_xml, 'w') as f:
           f.write (zap.core.xmlreport())
 
-      print ('FAIL: ' + str(fail_count) + '\tWARN: ' + str(warn_count) + '\tINFO: ' + str(info_count) +  
-        '\tIGNORE: ' + str(ignore_count) + '\tPASS: ' + str(pass_count))
+      print ('FAIL-NEW: ' + str(fail_count) + '\tFAIL-IP: ' + str(fail_inprog_count) + 
+        '\tWARN-NEW: ' + str(warn_count) + '\tWARN-IP: ' + str(warn_inprog_count) +
```

Comment:
I'm guessing that "IP" is meant to be 'in progress' but will that be confusing to others? (Need to be careful using IP in an IT context/world 😀 )